front/src/applications/stdcm/views/StdcmRequestModal.tsx

```diff
@@ -1,192 +1,23 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 
-import { cloneDeep } from 'lodash';
 import { useTranslation } from 'react-i18next';
 import ReactModal from 'react-modal';
-import { useSelector } from 'react-redux';
 
-import STDCM_REQUEST_STATUS from 'applications/stdcm/consts';
-import formatStdcmConf from 'applications/stdcm/formatStdcmConf';
-import type { StdcmRequestStatus, StdcmV2SuccessResponse } from 'applications/stdcm/types';
-import type { PostStdcmApiResponse, SimulationReport } from 'common/api/osrdEditoastApi';
-import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
 import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
 import { Spinner } from 'common/Loaders';
-import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
-import createTrain from 'modules/simulationResult/components/SpaceTimeChart/createTrain';
-import { CHART_AXES } from 'modules/simulationResult/consts';
-import { setFailure } from 'reducers/main';
-import type { OsrdStdcmConfState } from 'reducers/osrdconf/types';
-import {
-  updateConsolidatedSimulation,
-  updateSelectedTrainId,
-  updateSimulation,
-  updateSelectedProjection,
-} from 'reducers/osrdsimulation/actions';
-import type { Train } from 'reducers/osrdsimulation/types';
-import { getTrainScheduleV2Activated } from 'reducers/user/userSelectors';
-import { useAppDispatch } from 'store';
-import { castErrorToFailure } from 'utils/error';
 
-import { checkStdcmConf, formatStdcmPayload } from '../utils/formatStdcmConfV2';
-
-type StdcmRequestModalProps = {
-  setCurrentStdcmRequestStatus: (currentStdcmRequestStatus: StdcmRequestStatus) => void;
-  currentStdcmRequestStatus: StdcmRequestStatus;
-  setStdcmResults: (stdcmResults?: PostStdcmApiResponse) => void;
-  setStdcmV2Results: (stdcmV2Results: StdcmV2SuccessResponse | undefined) => void;
+export type StdcmRequestModalProps = {
+  isOpen: boolean;
+  cancelStdcmRequest: () => void;
 };
 
-const StdcmRequestModal = ({
-  setCurrentStdcmRequestStatus,
-  currentStdcmRequestStatus,
-  setStdcmResults,
-  setStdcmV2Results,
-}: StdcmRequestModalProps) => {
-  const dispatch = useAppDispatch();
+const StdcmRequestModal = ({ isOpen, cancelStdcmRequest }: StdcmRequestModalProps) => {
   const { t } = useTranslation(['translation', 'stdcm']);
```

Comment:
You don't use the translation file 'translation' so you can remove it and everywhere in the file, you can remove the 'stdcm:' before the translation key
```suggestion
  const { t } = useTranslation('stdcm');
```